Patches/Nakin Race/Nakin_Items.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class ="PatchOperationFindMod">
+		<mods>
+		<li>NakinRace</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+			<!-- Nakin Leather | values to represent the Race's natural toughness-->
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="Leather_Nakin"]/statBases/StuffPower_Armor_Sharp</xpath>
+					<value>
+						<StuffPower_Armor_Sharp>0.55</StuffPower_Armor_Sharp>

```

Comment:
Far too high. For comparison, humans have 0.05 sharp and 0.04 blunt. For a "tough" race, I'd give it 0.06 - 0.08 or so.